test/parser/repeat_parameter_sets_test.exs

```diff
@@ -0,0 +1,108 @@
+defmodule Membrane.H264.RepeatParameterSetsTest do
+  @moduledoc false
+
+  use ExUnit.Case
+
+  import Membrane.ChildrenSpec
+  import Membrane.Testing.Assertions
+  import Membrane.H264.Support.Common
+
+  alias Membrane.H264
+  alias Membrane.H264.Parser.NALuSplitter
+  alias Membrane.Testing.{Pipeline, Sink}
+
+  @in_path "../fixtures/input-30-240p-no-sps-pps.h264" |> Path.expand(__DIR__)
+  @ref_path "../fixtures/reference-30-240p-no-sps-pps.h264" |> Path.expand(__DIR__)
```

Comment:
I believe the `reference-30-240p-no-sps-pps.h264` contains SPS and PPS, so perhaps it might be worth renaming it